src/Appwrite/Template/Template.php

```diff
@@ -125,4 +176,36 @@ public static function fromCamelCaseToDash($input): string
     {
         return \str_replace([' ', '_'], '-', \strtolower(\preg_replace('/([a-zA-Z])(?=[A-Z])/', '$1-', $input)));
     }
+
+    /**
+     * setHtml
+     *
+     * Set the Raw html used by this template  
+     *
+     * @return string
+     * 
+     */
+    public function setHtml(string $html): self 
+    {
+        $this->html = $html;
+        return $this;
+    }
+
+     /**
+     * getHtml
+     *
+     * Get the Raw html that this template holds.  
+     *
+     * @return string
+     * 
+     */
+    public function getHtml(): string 
```

Comment:
Again as the above comment, not sure why we need dedicated setters as getters, seems like there is no much code repetitiveness this is saving us. Probably better avoid coupling with HTML, this template can be used with any view format.